drkafka/src/main/java/com/pinterest/doctorkafka/security/ExampleAuthorizationFilter.java

```diff
@@ -0,0 +1,68 @@
+package com.pinterest.doctorkafka.security;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.Priority;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.ext.Provider;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.pinterest.doctorkafka.config.DoctorKafkaConfig;
+
+import jersey.repackaged.com.google.common.collect.Sets;
+import jersey.repackaged.com.google.common.collect.Sets.SetView;
+
+/**
+ * This is a sample implementation of {@link DrKafkaAuthorizationFilter}
+ */
+@Provider
+@Priority(1000)
+public class ExampleAuthorizationFilter implements DrKafkaAuthorizationFilter {
```

Comment:
It can't be used by default since it follows certain conventions, any one attempting to enable authorization restrictions in Dr. Kafka needs to implement their own AuthorizationFilter since every environment is different.
The purpose of providing this class is to share with users how they can potentially implement their own AuthorizationFilter.

This PR doesn't change the default behavior of Dr. Kafka which is currently is no authorization restriction.

Reply:
Added the details in Dr. Kafka's docs https://github.com/pinterest/doctorkafka/pull/143/files#diff-ea40dcfd475c2164a28718b6d4c6d0ee